docs/file_changes.md

```diff
@@ -50,6 +50,15 @@ a directory to build. Tilt will watch the entire directory.
 Whenever a file in that directory changes, Tilt will re-build the image,
 then deploy any Kubernetes resources that depend on that image.
 
+### Custom and Local Resource Builds
+
+When you build other types of resources (like `custom_build()` and
+`local_resource()`), you can specify a directory or a set of directories
```

Comment:
here and below: deps can be files

Reply:
i changed it to 'paths', which i think most people will understand to be files or directories